docs/Introduction.md

```diff
@@ -79,4 +80,4 @@ There are also related apps which don’t really overlap in functionality, but t
 
 ## License
 
-AltTab is under the [GPL-3.0 license](https://github.com/lwouis/alt-tab-macos/blob/master/LICENCE.md). 
+AltTab is under the [GPL-3.0 license](https://github.com/lwouis/alt-tab-macos/blob/master/LICENCE.md).
```

Comment:
Could you please remove that change from the commit? Thank you~

Reply:
Thank you.
Removed -> https://github.com/lwouis/alt-tab-macos/pull/1660/commits/253a73cb896aff2d36de338a23fb5f585b67eb1b